core/src/main/java/org/elasticsearch/index/mapper/KeywordFieldMapper.java

```diff
@@ -92,11 +97,17 @@ public Builder eagerGlobalOrdinals(boolean eagerGlobalOrdinals) {
             return builder;
         }
 
+        public Builder normalizer(NamedAnalyzer normalizer) {
+            fieldType.setSearchAnalyzer(normalizer);
+            this.normalizer = normalizer;
+            return builder;
+        }
```

Comment:
It seems that the normalizer can be changed when the field is updated ? It should be prohibited because we apply the normalizer at index and search time. Though the check is done on the index analyzer only and here we set the search analyzer so it won't prevent the normalizer to be changed at runtime.